wordpress-mobile/WordPress-Android
# Show the correct error message when reply to comment fails

Fixes #3977 

To test: 
- Go to the site Comments section
- Select a comment from a closed thread
- Try to reply and get a "Reply failed" toast

~~Needs review: @roundhill~~

The error message returned from the server is shown to the user in case of `XMLRPCFault` only (that means when the server replies correctly with a XML-RPC response document). Default error message is shown when IOErrors, or any other problem.
Errors are always shown to the user in case of replying to a comment by using the the REST layer.
